autoload/neoformat/formatters/sh.vim

```diff
@@ -3,9 +3,11 @@ function! neoformat#formatters#sh#enabled() abort
 endfunction
 
 function! neoformat#formatters#sh#shfmt() abort
+    let opts = get(g:, 'shfmt_opt', '')
+
     return {
             \ 'exe': 'shfmt',
-            \ 'args': ['-i ' . shiftwidth()],
+            \ 'args': ['-i ' . shiftwidth(), opts],
```

Comment:
should we add a space `' '` between the `shiftwidth()` and `opts`

Reply:
I was assuming that was the behaviour of the array. Am I wrong? It seemd to work as expected in my tests...

Reply:
That being said: if that's the case, we could rewrite that to `['-i', shiftwidth(), opts]`

Reply:
Oh I think you're right